src/ModelContextProtocol.AspNetCore/HttpServerTransportOptions.cs

```diff
@@ -22,6 +22,13 @@ public class HttpServerTransportOptions
     /// </summary>
     public Func<HttpContext, IMcpServer, CancellationToken, Task>? RunSessionHandler { get; set; }
 
+    /// <summary>
+    /// Gets or sets whether the server should run in a stateless mode that does not require all requests for a given session
+    /// to arrive to the same ASP.NET Core application process. If true, the /sse endpoint will be disabled, and
+    /// client capabilities will be round-tripped as part of the mcp-session-id header instead of stored in memory. Defaults to false.
+    /// </summary>
```

Comment:
```suggestion
    /// <summary>
    /// Gets or sets whether the server should run in a stateless mode that does not require all requests for a given session
    /// to arrive to the same ASP.NET Core application process.
    /// </summary>
    /// <remarks>
    /// If <see langword="true"/>, the "/sse" endpoint will be disabled, and
    /// client capabilities will be round-tripped as part of the "mcp-session-id" header instead of stored in memory.
    /// Defaults to <see langword="false"/>.
    /// </remarks>
```